cognee/api/v1/cognify/code_graph_pipeline.py

```diff
@@ -38,12 +55,37 @@ async def run_code_graph_pipeline(repo_path):
     await cognee.prune.prune_system(metadata=True)
     await create_db_and_tables()
 
+    embedding_engine = get_embedding_engine()
+    cognee_config = get_cognify_config()
+    user = await get_default_user()
+
     tasks = [
         Task(get_repo_file_dependencies),
-        Task(enrich_dependency_graph, task_config={"batch_size": 50}),
+        Task(enrich_dependency_graph),
         Task(expand_dependency_graph, task_config={"batch_size": 50}),
+        Task(get_source_code_chunks, embedding_model=embedding_engine.model, task_config={"batch_size": 50}),
```

Comment:
Where does `get_source_code_chunks` come from?

Reply:
hey @borisarzentar I messed up the commits on this PR, moved the meaningful part here: https://github.com/topoteretes/cognee/pull/395

I am closing this one now, sorry for the confusion